File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self,nums):

        n = len(nums)
        sum_all = (n * (n + 1) // 2)

        sum_given = sum(nums)

        missing_number = sum_all - sum_given

        return missing_number
    
obj = Solution()
    
given_array = [3,0,1]
result = obj.missingNumber(given_array)

print("The missing number is: ",result)
        

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int s1 = (n*(n+1))/2;
        int s2 = 0;
        for (int i: nums) {
            s2 += i;
        }

        return s1-s2;
    }
}